pages/docs/tutorials/streetlights.md

```diff
@@ -1,244 +1,254 @@
+{`npm install -g @asyncapi/generator`}
+</CodeBlock>
+
+### 2. Create a directory for your projects and enter it:
+<CodeBlock language="bash">
+{`mkdir streetlights && cd "$_"`}
+</CodeBlock>
+
+### 3. Create a file with the AsyncAPI machine-readable description you defined before using terminal. The `cat` command is a utility command in Linux. On Windows use `type` instead of `cat`:
+<CodeBlock language="yaml">
+{`cat <<EOT >> asyncapi.yaml
+asyncapi: '2.2.0'
+info:
+  title: Streetlights API
+  version: '1.0.0'
+  description: |
+    The Smartylighting Streetlights API allows you
+    to remotely manage the city lights.
+  license:
+    name: Apache 2.0
+    url: 'https://www.apache.org/licenses/LICENSE-2.0'
+servers:
+  mosquitto:
+    url: mqtt://test.mosquitto.org
+    protocol: mqtt
+channels:
+  light/measured:
+    publish:
+      summary: Inform about environmental lighting conditions for a particular streetlight.
+      operationId: onLightMeasured
+      message:
+        name: LightMeasured
+        payload:
+          type: object
+          properties:
+            id:
+              type: integer
+              minimum: 0
+              description: Id of the streetlight.
+            lumens:
+              type: integer
+              minimum: 0
+              description: Light intensity measured in lumens.
+            sentAt:
+              type: string
+              format: date-time
+              description: Date and time when the message was sent.
+EOT`}
+</CodeBlock>
+
+### 4. Trigger generation of the Node.js code:
+<CodeBlock language="bash">
+{`ag asyncapi.yaml @asyncapi/nodejs-template -o output -p server=mosquitto`}
+</CodeBlock>
+
+### 5. And voilà! List all files in directory and check that Node.js application is generated:
+<CodeBlock language="bash">
+{`cd output && ls`}
+</CodeBlock>
+
+# Running your code
+
+### 1. Install dependencies of newly generated application:
+<CodeBlock language="bash">
+{`npm install`}
+</CodeBlock>
+
+### 2. Start the application:
+<CodeBlock language="bash">
+{`npm start`}
+</CodeBlock>
+
+### 3. In another terminal start MQTT broker:
+<CodeBlock language="bash">
+{`docker run -it -p 1883:1883 eclipse-mosquitto:1.5`}
+</CodeBlock>
+
+### 4. In another terminal install the MQTT.js library:
+<CodeBlock language="bash">
+{`npm install mqtt -g`}
+</CodeBlock>
+
+### 5. Send a correct message to your application:
+<CodeBlock language="bash">
+{`mqtt pub -t 'light/measured' -h 'test.mosquitto.org' -m '{"id": 1, "lumens": 3, "sentAt": "2017-06-07T12:34:32.000Z"}'`}
+</CodeBlock>
+
+### 6. Send an incorrect message to your application:
+<CodeBlock language="bash">
+{`mqtt pub -t 'light/measured' -h 'test.mosquitto.org' -m '{"id": 1, "lumens": "3", "sentAt": "2017-06-07T12:34:32.000Z"}'`}
+</CodeBlock>
+
+### 7. Go back to the previous terminal and check if your application logged the streetlight condition you just sent, with errors related to the invalid message.
+
+# Summary

```

Comment:
Hey @starlightknown! 😸 So remember that for the **Summary** section in our tutorials, we are writing:
- a summary of what the user just built and what technologies were used (again)
- an explanation of what the user learned to do 

A big part of improving this tutorial means both re-writing and adding new content ❤️ based on our new IA goals.

Reply:
made changes